recipe/meta.yaml

```diff
@@ -17,7 +17,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 3
+  number: 0
+  skip: True  # [unix and not py35]
```

Comment:
I've added this as we don't need to build for different versions of python in unix. Python is merely used for running the test suite.